frontend/src/metabase/parameters/components/ParameterValueWidget.tsx

```diff
@@ -68,17 +68,36 @@ export const ParameterValueWidget = ({
   const [isFocused, setIsFocused] = useState(false);
 
   const hasValue = !parameterHasNoDisplayValue(value);
+  const fieldHasValueOrFocus = parameter.value != null || isFocused;
   const noPopover = hasNoPopover(parameter);
   const parameterTypeIcon = getParameterIconName(parameter);
   const showTypeIcon = !isEditing && !hasValue && !isFocused;
 
   const [isOpen, { close, toggle }] = useDisclosure();
 
   const getOptionalActionIcon = () => {
+    const { default: defaultValue } = parameter;
+
+    if (
+      defaultValue &&
```

Comment:
Could check for `!= null` here? I haven't checked yet but it could create issues with `0` values

Reply:
Done, updated all occurrences in this file: 3ac0bbe97b2ea1a96e5a06938523a0e271e90be7

----

FYI:

I have never encountered a number here (numbers come as strings, including zeros).
As far as I checked the value can be:
- a string
- an array of strings
- an array of numbers

Reply:
yeah I think we need to treat empty strings as `null` here. Not sure what the cleanest way to do that

Reply:
Done: 316a3ff93ed240c54437d74df6e1e19e2d818c85

I used existing `parameterHasNoDisplayValue` and also made sure it works with zeros.